presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -8853,4 +8853,52 @@ public void testAggregationPushedBelowOuterJoin()
                         "GROUP BY v1.col1",
                 "VALUES 24");
     }
+
+    @Test
+    public void testLateralJoin()
```

Comment:
Can you add these tests in the commit(s) that enable them? (assuming that's a result each of the optimizers introduced above). It will make it easier to track and relate the functionality changes to their corresponding tests.